addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -244,6 +244,7 @@
                                                 'dynamic_placeholder': true,
                                                 'cssReadonly': 'mass_mailing.iframe_css_assets_edit'
                                         }" attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}"/>
+                                        <field name="body_html" invisible="1"/>
```

Comment:
Nitpick, but it seems like we usually put invisible fields above the place they're used.